Apps/ValidationTests/Win32/App.cpp

```diff
@@ -64,7 +80,6 @@ namespace
         }
 
         runtime.reset();
-        Babylon::Plugins::NativeEngine::DeinitializeGraphics();
```

Comment:
Does this need to be called from somewhere? It looks like in this PR it's removed (presumably because it was causing problems) and never put back, unless I missed it somewhere.